fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/CascadesPlanner.java

```diff
@@ -426,8 +432,7 @@ private void planPartial(@Nonnull Supplier<Reference> referenceSupplier,
 
                     if (logger.isTraceEnabled()) {
                         logger.trace(KeyValueLogMessage.of("planner state",
-                                "taskStackSize", taskStack.size(),
-                                "memo", new ReferencePrinter(currentRoot)));
+                                "taskStackSize", taskStack.size()));
```

Comment:
Got rid of the `ReferencePrinter` class. I think it's useless to see the expression DAG (with variants) stringified in a log. And it can be waaaaay to big.